src/poset/badger_store.go

```diff
@@ -51,9 +52,6 @@ func NewBadgerStore(participants *peers.Peers, cacheSize int, path string) (*Bad
 	if err := store.dbSetRoots(inmemStore.rootsByParticipant); err != nil {
 		return nil, err
 	}
-	if err := store.dbSetRootEvents(inmemStore.rootsByParticipant); err != nil {
```

Comment:
dbSetRootEvents() put leaf events into database, without these events we have a number of error messages like these:
WARN[0010] failed to get parent: Event, Root4002739731, Not Found  id=4002739731
WARN[0010] failed to get other parent: Key cannot be empty  id=4002739731
Pinging stopped after  10  iterations
WARN[0010] failed to get parent: Event, Root3892269912, Not Found  id=3892269912
WARN[0010] failed to get other parent: Key cannot be empty  id=3892269912
Pinging stopped after  10  iterations
WARN[0010] failed to get parent: Event, Root2806285528, Not Found  id=2806285528
WARN[0010] failed to get other parent: Key cannot be empty  id=2806285528
WARN[0010] failed to get parent: Event, Root1315121877, Not Found  id=1315121877
WARN[0010] failed to get other parent: Key cannot be empty  id=1315121877
WARN[0010] failed to get parent: Event, Root1215690256, Not Found  id=1215690256
WARN[0010] failed to get other parent: Key cannot be empty  id=1215690256
WARN[0010] failed to get parent: Event, Root3811045049, Not Found  id=3811045049
WARN[0010] failed to get other parent: Key cannot be empty  id=3811045049
WARN[0010] failed to get parent: Event, Root370705123, Not Found  id=370705123
WARN[0010] failed to get other parent: Key cannot be empty  id=370705123
WARN[0010] failed to get parent: Event, Root1657001824, Not Found  id=1657001824
WARN[0010] failed to get other parent: Key cannot be empty  id=1657001824
WARN[0010] failed to get parent: Event, Root1318802166, Not Found  id=1318802166

Failure to get parent should be considered as an incident of attack on the network and preventing event processing, not allowing slip through as it's currently implemented.

However, these leaf event have another drawback: they have only one signature instead of two required. This might be rectified by introducing a master key (as part of an algorithm) used only as a second private key for signing leaf events in creation of a new database.

Thus adding leaf events into database signed with two keys will make new database creation correct.

Reply:
It was already implemented in a separate PR, in which tests where adjusted for it.
Why this PR needs to removed it, is anything else broken?